packages/atomic/cypress/integration/result-list/result-components/result-multi-value-text.cypress.ts

```diff
@@ -79,7 +87,7 @@ describe('Result MultiValueText Component', () => {
         .init();
     });
 
-    assertRemovesComponent(ResultMultiValueTextSelectors.shadow);
+    assertRemovesComponent(() => cy.get(resultMultiValueTextComponent));
```

Comment:
These selectors were entering the shadowDOM of elements, which means that Cypress asserted that these elements exist in order to access their shadowDOM before asserting that they don't.